src/jsd/actuator.h

```diff
@@ -60,6 +60,13 @@ class Actuator : public JsdDeviceBase
   void      Reset() override;
   bool      SetOutputPosition(double position);
   bool      HasAbsoluteEncoder();
+  ActuatorStateMachineState GetActuatorStateMachineState();
```

Comment:
fine, I don't hate it but it is already contained in the `GetState()` struct

Reply:
`GetState()` returns `DeviceState state_`, which is a separate from `ActuatorStateMachineState actuator_sms`